setup.py

```diff
@@ -60,9 +60,9 @@
     tests_require=TESTS_REQUIRES,
     extras_require=EXTRAS,
     packages=['kubernetes', 'kubernetes.client', 'kubernetes.config',
-              'kubernetes.watch', 'kubernetes.client.api',
-              'kubernetes.stream', 'kubernetes.client.models',
-              'kubernetes.utils', 'kubernetes.client.apis',
+              'kubernetes.watch', 'kubernetes.client.api', 'kubernetes.client.api_client'
+              'kubernetes.client.rest', 'kubernetes.stream', 'kubernetes.client.models',
+              'kubernetes_asyncio', 'kubernetes.utils', 'kubernetes.client.apis',
```

Comment:
It's unclear to me what issue this PR is addressing. Could you elaborate?

Reply:
Aims to provide various functionalities necessary for interacting with Kubernetes clusters using the Kubernetes Python client libraries

Reply:
`kubernetes_asyncio` is not part of this repo. I think you can include it separately in your own environment?

Reply:
> `kubernetes_asyncio` is not part of this repo. I think you can include it separately in your own environment?

However it's a python package which supports both core Kubernetes API resources and custom resource definitions (CRDs), making it a versatile tool for various Kubernetes-related tasks.